packages/react-server/src/ReactFizzServer.js

```diff
@@ -1409,6 +1499,8 @@ function abortTask(task: Task): void {
 
     if (!boundary.forceClientRender) {
       boundary.forceClientRender = true;
+      boundary.errorMessage =
+        'This Suspense boundary was aborted by the server';
```

Comment:
For renderToString we send this for every suspense boundary since that works by aborting early.

This seems a little long for production behavior, especially if we've already timed out we might want to keep it short.

I guess the reason we do this is because we don't to treat this as an error and call onError so we don't have a hash for it. Should we treat this as an error though? Maybe you want to know about what parts got aborted.

We could also maybe just assume on the client that if we don't have any hash (like undefined/missing attribute as opposed to empty string) that it was aborted on the server?

Reply:
>assume on the client that if we don't have any hash ... that it was aborted on the server?

hmm, if you don't use onError on the server the hash is going to be null (currently) and that doesn't get emitted on the template or render instruction. We'd have to distinguish between does not exist (we didn't call onError) and exists but empty (user did not set up onError) and would lead to extra attributes (data-hash) on every non-abort error. This feels fragile too because it would fall apart the moment we had a different kind of error case not requiring a user onError.

maybe there is a different kind of thing like `data-sig(nal)` that uses an enum for things like abort? it'd make the instruction scripts a bit longer but that's amortized over each client rendered boundary